app/helpers/foreman_openscap_helper.rb

```diff
@@ -15,10 +15,15 @@ def scap_doc_url(section = '')
     documentation_url(section, root_url: scap_root_url)
   end
 
+  def doc_flavor
+    ForemanOpenscap.with_katello? ? 'katello' : 'foreman'
+  end
+
   def scap_root_url
     @scap_root_url ||= begin
-      version = ForemanOpenscap::VERSION.split('.')[0..-2].join('.')
-      "https://theforeman.org/plugins/foreman_openscap/#{version}/index.html"
+      version = SETTINGS[:version]
+      version = version.tag == 'develop' ? 'nightly' : version.short
+      "https://docs.theforeman.org/#{version}/Managing_Security_Compliance/index-#{doc_flavor}-el.html#"
```

Comment:
```suggestion
      "https://docs.theforeman.org/#{version}/Managing_Security_Compliance/index-#{doc_flavor}.html#"
```

Reply:
Not sure I understand...
[Peek 2023-10-19 15-20.webm](https://github.com/theforeman/foreman_openscap/assets/32508194/589b01e8-d03d-4b8d-b4ea-3adefc9bf16f)

Reply:
Oh, right. Because katello doesn't run on non-el, but foreman does. Will fix